src/StackExchange.Metrics/Handlers/StatsdMetricHandler.cs

```diff
@@ -307,6 +307,7 @@ protected override void SerializeMetric(IBufferWriter<byte> writer, in MetricRea
                             "Span was not big enough to write metric value"
                         );
 
+                        ex.Data.Add("Name", reading.Name.ToString());
```

Comment:
Nit: No need to `ToString` here, it's already a string

Reply:
Too much ctrl+v, not enough ctrl+x 😄